.travis.yml

```diff
@@ -2,12 +2,25 @@
 # * http://www.objc.io/issue-6/travis-ci.html
 # * https://github.com/supermarin/xcpretty#usage
 
+osx_image: xcode 8.3
 language: objective-c
-# cache: cocoapods
-# podfile: Example/Podfile
-# before_install:
-# - gem install cocoapods # Since Travis is not always on latest version
-# - pod install --project-directory=Example
+cache:
+  - bundler
+  - cocoapods
+podfile: Example/Podfile
+before_install:
+ - gem install cocoapods # Since Travis is not always on latest version
+ - gem install xcpretty -N
+ - pod update --project-directory=Example
+
 script:
 - set -o pipefail && xcodebuild test -workspace Example/Heartland-iOS-SDK.xcworkspace -scheme Heartland-iOS-SDK-Example -sdk iphonesimulator ONLY_ACTIVE_ARCH=NO | xcpretty
 - pod lib lint
```

Comment:
With `pod update` having the `project-directory` option being passed, I wanted to verify that it's not needed for `pod lib lint` and `pod trunk push`.